packages/twenty-server/src/engine/metadata-modules/remote-server/dtos/create-remote-server.input.ts

```diff
@@ -0,0 +1,31 @@
+import { Field, InputType } from '@nestjs/graphql';
+
+import { IsString } from 'class-validator';
+
+@InputType()
+export class CreateRemoteServerInput {
+  @Field(() => String)
+  @IsString()
+  host: string;
+
+  @Field(() => String)
+  @IsString()
+  port: string;
+
+  @Field(() => String)
+  @IsString()
+  database: string;
+
+  @Field(() => String)
+  @IsString()
+  username: string;
+
+  @Field(() => String)
+  @IsString()
+  password: string;
+
+  // TODO: Decide if this should be added during the creation
```

Comment:
@charlesBochet could we have several schema?

Reply:
I guess you had this discussion together already but are you sure you want to store password, username, etc? I didn't include it in the proposed schema because it didn't seem like it was a requirement and increases security risk + desynchronization risk. Why did you decide to go this way?

Reply:
Also I had changed schema to a json column because it's not always a schema, the key/value(s) you need to pass depend on the wrapper type

Reply:
In the design, we want to display possible tables that the user could choose to sync. Foreign data wrappers do not allow to read the foreign schema. You need to import it and to already know the name.
We will need to connect to the foreign database directly and read all available schemas. Same to know if a schema has been updated